src/main/java/seedu/address/model/module/Module.java

```diff
@@ -17,6 +19,7 @@
 public class Module {
     private final ModuleName name;
     private final ZoomLink zoomLink;
+    private Map<String, ZoomLink> zoomLinks;
```

Comment:
Perhaps this field could be declared final to ensure immutability.

Reply:
Yes, I will update it as final. Just want to make sure first that the HashMap methods are correct before editing the rest of the constructors 😅